libensemble/tests/regression_tests/test_persistent_gp.py

```diff
@@ -32,6 +32,8 @@
 
 # Dragonfly uses a deprecated np.asscalar command.
 warnings.filterwarnings("ignore", category=DeprecationWarning)
+# Dragonfly uses a deprecated np.object scalar command.
```

Comment:
Double check that this is needed (given that you've fixed dragonfly on your fork). Otherwise, looks great!